gulpfile.js/copy.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
This and a few other of the JS files are missing the file header including license information, which need to be present.

Reply:
Added.